src/components/Dashboard/DeviceFooter.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { Card, Col, Row } from 'react-bootstrap';
+
+import styles from './DeviceFooter.scss';
+
+type Props = {
+    linkquality?: number;
+    battery?: number;
+    voltage?: number;
+    consumption?: number;
+    temperature?: number;
+    lastUpdate: number;
+};
+
+class DeviceFooter extends Component<Props> {
+    static defaultProps = {
+        voltage: 0,
+    };
+
+    render() {
+        const { linkquality, battery, voltage, consumption, temperature, lastUpdate } = this.props;
+
+        let batteryIcon = 'fa-battery-empty';
+        if (battery) {
+            if (battery >= 95) {
+                batteryIcon = 'fa-battery-full';
+            } else if (battery >= 75) {
+                batteryIcon = 'fa-battery-three-quarters';
+            } else if (battery >= 50) {
+                batteryIcon = 'fa-battery-half';
+            } else if (battery >= 10) {
+                batteryIcon = 'fa-battery-quarter';
+            }
+        }
+
+        const outdated = (Date.now() - lastUpdate) / 1000 >= 3600;
```

Comment:
last_seen can be different in formats ISO_8601, ISO_8601_local, epoch
Do refere to `utils.tsx/lastSeen`